web/src/emoji_picker.js

```diff
@@ -346,6 +372,7 @@ function reset_emoji_showcase() {
 function update_emoji_showcase($focused_emoji) {
     // Don't use jQuery's data() function here. It has the side-effect
     // of converting emoji names like :100:, :1234: etc to number.
+    $(".emoji-popover .simplebar-content-wrapper").attr("tabindex", "-1");
```

Comment:
Can you add a comment here for the purpose of this? I assume this is to avoid focus on the simplebar wrapper.